log/log.go

```diff
@@ -315,6 +316,17 @@ func (log *impl) Infof(format string, a ...interface{}) {
 	log.Info(fmt.Sprintf(format, a...))
 }
 
+// InfoObject logs an INFO level JSON-serialized object message.
+func (log *impl) InfoObject(msg string, obj interface{}) {
+	jsonObj, err := json.Marshal(obj)
+	if err != nil {
+		log.auditAtLevel(syslog.LOG_ERR, fmt.Sprintf("Object could not be serialized to JSON. Raw: %+v", obj))
```

Comment:
Seems like it would be helpful to still include the originally-intended `msg` in here somehow.

Reply:
I can agree with this. I'll do a followup PR which includes the `msg` for both `AuditObject()` and `InfoObject()`.

https://github.com/letsencrypt/boulder/blob/5c49231ea6bffe0b51e528944724f58f6df75eca/log/log.go#L340-L350

Reply:
https://github.com/letsencrypt/boulder/pull/6650